<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWindow</class>
 <widget class="QWidget" name="ConfigWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration des sondes</string>
  </property>

  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="sondeList">
     <property name="toolTip">
      <string>Liste des sondes configur√©es</string>
     </property>
    </widget>
   </item>

   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelId">
       <property name="text">
        <string>ID de la sonde</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="idEdit"/>
     </item>

     <item row="1" column="0">
      <widget class="QLabel" name="labelName">
       <property name="text">
        <string>Nom de la sonde</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nameEdit"/>
     </item>

     <item row="2" column="0">
      <widget class="QLabel" name="labelIp">
       <property name="text">
        <string>Adresse IP</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="ipEdit"/>
     </item>

     <item row="3" column="0">
      <widget class="QLabel" name="labelPort">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="portEdit"/>
     </item>

     <item row="4" column="0">
      <widget class="QLabel" name="labelMqttUser">
       <property name="text">
        <string>Utilisateur MQTT</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="mqttUserEdit"/>
     </item>

     <item row="5" column="0">
      <widget class="QLabel" name="labelMqttPassword">
       <property name="text">
        <string>Mot de passe MQTT</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="mqttPasswordEdit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>

     <item row="6" column="0">
      <widget class="QLabel" name="labelMqttTopic">
       <property name="text">
        <string>Topic MQTT</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="mqttTopicEdit"/>
     </item>
    </layout>
   </item>

   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Ajouter Sonde</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeButton">
       <property name="text">
        <string>Supprimer Sonde</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="configureButton">
       <property name="text">
        <string>Configurer Sonde</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>